src/components/import/badges/Badge.scss

```diff
@@ -0,0 +1,17 @@
+.ocs-badge {
+  &__inline {
+    background-color: var(--pf-global--BackgroundColor--200);
+    border: 1px solid var(--pf-global--BorderColor--light-100);
+  }
+}
+
+.ocs-preview-badge {
```

Comment:
Think we could get rid of the ocs prefix and use our own prefix

Reply:
Sure, will fix it in [GITOPS-2291](https://issues.redhat.com/browse/GITOPS-2291).